lib/types/p2p.ts

```diff
@@ -21,7 +23,7 @@ export type HandshakeState = {
   lndltcPubKey?: string;
 };
 
-export function isHandshakeState(obj: any): obj is HandshakeState {
```

Comment:
I know we don't use this method currently but I *think* we may use it when it comes to validating that a HELLO packet we've received is valid, so I don't think I'd remove it yet.